### PR TITLE
MINOR: Move upgraded docs from site to Kafka docs

### PR DESCRIPTION
**Pull request by mimaison**
A number of doc updates have been applied to the website. For the 2.6.1 release, resync Kafka's docs folder with the website.

### Committer Checklist (excluded from commit message)
- [ ] Verify design and implementation 
- [ ] Verify test coverage and CI build status
- [ ] Verify documentation (including upgrade notes)

### CODE REVIEW

**Comment by mimaison** [comment]
Copied docs from the 26 folder in kafka-site. I checked and as far as I can tell, this already included all recent changes made to `/docs` in the 2.6 branch

**Comment by mimaison** [comment]
@bbejeck  Can you take a look?

**Comment by bbejeck** [comment]
@mimaison ack, I'll take a look

File: docs/js/templateData.js [hunk 1]
```diff
@@ -19,6 +19,6 @@ limitations under the License.
 var context={
     "version": "26",
     "dotVersion": "2.6",
-    "fullDotVersion": "2.6.1",
+    "fullDotVersion": "2.6.0",
```
Comment (bbejeck) [inline docs/js/templateData.js]: this needs to be `2.6.1`

File: docs/ops.html [hunk 2]
```diff
@@ -538,98 +472,74 @@ <h5>Safe usage of throttled replication</h5>
   <p><i>(2) Ensuring Progress:</i></p>
   <p>If the throttle is set too low, in comparison to the incoming write rate, it is possible for replication to not
       make progress. This occurs when:</p>
-  <pre>max(BytesInPerSec) > throttle</pre>
+  <pre>max(BytesInPerSec) > throttle</code></pre>
```
Comment (bbejeck) [inline docs/ops.html]: this is missing an opening `<code>`

File: docs/ops.html [hunk 3]
```diff
@@ -538,98 +472,74 @@ <h5>Safe usage of throttled replication</h5>
...
   <p>
       Where BytesInPerSec is the metric that monitors the write throughput of producers into each broker. </p>
   <p>The administrator can monitor whether replication is making progress, during the rebalance, using the metric:</p>
 
-  <pre>kafka.server:type=FetcherLagMetrics,name=ConsumerLag,clientId=([-.\w]+),topic=([-.\w]+),partition=([0-9]+)</pre>
+  <pre>kafka.server:type=FetcherLagMetrics,name=ConsumerLag,clientId=([-.\w]+),topic=([-.\w]+),partition=([0-9]+)</code></pre>
```
Comment (bbejeck) [inline docs/ops.html]: same here

File: docs/ops.html [hunk 4]
```diff
@@ -1653,9 +1559,9 @@ <h4><a id="kafka_streams_monitoring" href="#kafka_streams_monitoring">Streams Mo
   Use the following configuration option to specify which metrics
   you want collected:
 
-<pre>metrics.recording.level="info"</pre>
+<pre>metrics.recording.level="info"</code></pre>
```
Comment (bbejeck) [inline docs/ops.html]: here too

File: docs/security.html [hunk 5]
```diff
@@ -316,28 +294,24 @@ <h5>Host Name Verification</h5>
...
+                    <pre class="line-numbers"><code class="language-bash">                        openssl x509 -in certificate.crt -text -noout</code></pre>
                 </li>
             </ol>
         </li>
-        <li><h4><a id="security_configbroker" href="#security_configbroker">Configuring Kafka Brokers</a></h4>
+        <li><h4 class="anchor-heading"><a id="security_configbroker" class="anchor-link"></a><a href="#security_configbroker">Configuring Kafka Brokers</a></h4>
             Kafka Brokers support listening for connections on multiple ports.
             We need to configure the following property in server.properties, which must have one or more comma-separated values:
-            <pre>listeners</pre>
+            <pre>listeners</code></pre>
```
Comment (bbejeck) [inline docs/security.html]: ditto

File: docs/upgrade.html [hunk 6]
```diff
@@ -39,17 +81,51 @@ <h5><a id="upgrade_260_notable" href="#upgrade_260_notable">Notable changes in 2
         Fetch requests and other requests intended only for the leader or follower return NOT_LEADER_OR_FOLLOWER(6) instead of REPLICA_NOT_AVAILABLE(9)
         if the broker is not a replica, ensuring that this transient error during reassignments is handled by all clients as a retriable exception.
     </li>
-    <li>There are several notable changes to the reassignment tool <code>kafka-reassign-partitions.sh</code>
```
Comment (bbejeck) [inline docs/upgrade.html]: This section should go back in I believe

File: docs/upgrade.html [hunk 7]
```diff
@@ -19,7 +19,49 @@
 
 <script id="upgrade-template" type="text/x-handlebars-template">
 
-<h5><a id="upgrade_260_notable" href="#upgrade_260_notable">Notable changes in 2.6.0</a></h5>
+<h4><a id="upgrade_2_6_0" href="#upgrade_2_6_0">Upgrading to 2.6.0 from any version 0.8.x through 2.5.x</a></h4>
```
Comment (bbejeck) [inline docs/upgrade.html]: I think `Upgrading to 2.6.0`  needs to go to `Upgrading to 2.6.1`

File: docs/quickstart-zookeeper.html [hunk 8]
```diff
@@ -0,0 +1,277 @@
...
+
+<script id="quickstart-template" type="text/x-handlebars-template">
+
+      <div class="quickstart-step">
+      <h4 class="anchor-heading">
+          <a class="anchor-link" id="quickstart_download" href="#quickstart_download"></a>
+          <a href="#quickstart_download">Step 1: Get Kafka</a>
+      </h4>
+
+      <p>
```
Comment (bbejeck) [inline docs/quickstart-zookeeper.html:30]: refs in this file to `2.6.0` need to be `2.6.1`

**Review by bbejeck — COMMENTED** [review]
Thanks for the PR @mimaison over all LGTM, but there's a few places where the version needs to change to `2.6.1`

**Comment by mimaison** [comment]
Thanks @bbejeck ! I've pushed an update

**Review by bbejeck — APPROVED** [review]
Thanks for the update @mimaison. LGTM

**Pull request merged**